libkbfs/folder_branch_ops.go

```diff
@@ -1723,13 +1723,16 @@ func (fbo *folderBranchOps) syncBlockAndCheckEmbedLocked(ctx context.Context,
 		return path{}, DirEntry{}, nil, err
 	}
 
-	// do the block changes need their own blocks?
-	bsplit := fbo.config.BlockSplitter()
-	if !bsplit.ShouldEmbedBlockChanges(&md.data.Changes) {
-		err = fbo.unembedBlockChanges(ctx, bps, md, &md.data.Changes,
-			uid)
-		if err != nil {
-			return path{}, DirEntry{}, nil, err
+	// Do the block changes need their own blocks?  Unembed only if
```

Comment:
The specific reason for this change here was that `ShouldEmbedBlockChanges` was being called twice,  and after the first one the MD was still being changed, which resulted in a panic (because the sizeEstimate wasn't 0).  So anywhere else that happens, the panic should catch it.

However, your suggestion seems like a good check (though unrelated to this code snippet), so I'll add it.